### PR TITLE
Refactor Ads

### PR DESCRIPTION
**Pull request by leggomuhgreggo**
The primary focus of this task is to update the ad components to resolve the minHeight and minWidth from a mapping object with the ad size specs in constants, based on the ad component's type. 

I considered that this could be included in the HOC used to make ad components, and refactored the existing code to include it and also simplified the logic to generate an ad component based on type.

Wasn't 100% sure on some things so let me know if there's anything I overlooked that'd interfere with the requirements for the ad delivery code. 

Incidental change:
Updating component path for builds to the docker instance

### CODE REVIEW

File: src/components/ads/ad-utils/pbhAdFunctions.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const registerAdFunctionWithId = (adFunc, adId) => {
+  getAdLoadArray().push(() => adFunc(adId));
+};
+
+/**
+ * getAdLoadArray
+ *
+ * Functional wrapper to resolve PbhAdUnit_load array from window
+ */
+const getAdLoadArray = () => window.PbhAdUnit_cmd || [];
```
Comment (kitwestneat) [inline src/components/ads/ad-utils/pbhAdFunctions.js]: (major) typo

File: src/components/ads/ad-utils/pbhAdFunctions.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+/**
+ * startAds
+ *
+ * Appends pbh_start_ads to the end of the PbhAdUnit_load queue array.
+ * Used in the App component's componentDidMount method
+ */
+export const startAds = () => {
+  getAdLoadArray().push(() => window.pbh_start_ads());
```
Comment (kitwestneat) [inline src/components/ads/ad-utils/pbhAdFunctions.js]: (minor) if you are wrapping the arrays anyway, what do you think about wrapping the push as well, so it's just something like addLoadCallback(fn) and addInitCallback(fn)

File: src/components/ads/ad-utils/pbhAdFunctions.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+ * getAdRegistrationID
+ *
+ * Registers instance of PBH ad function for given ad type and returns the
+ * associated ID. Used in the MakeAdComponent HOC.
+ *
+ * @param {String} adType - the type of add from the AD_TYPE enum in constants
+ * @returns {String} id
+ */
+export const getAdRegistrationID = adType => {
+  const adFunc = getCreateAdFuncForType(adType);
```
Comment (kitwestneat) [inline src/components/ads/ad-utils/pbhAdFunctions.js]: (minor) can you rename getAdRegistrationID to make the side effect more obvious?

Comment (leggomuhgreggo) [reply src/components/ads/ad-utils/pbhAdFunctions.js]: Agree with thinking but deferring refactor.

File: src/components/ads/ad-utils/pbhAdFunctions.js (hunk 1 above)
Comment (kitwestneat) [reply src/components/ads/ad-utils/pbhAdFunctions.js]: The typo wasn't in the doc...

Comment (leggomuhgreggo) [reply src/components/ads/ad-utils/pbhAdFunctions.js]: #80

**Pull request merged**